src/main/java/seedu/zerotoone/MainApp.java

```diff
@@ -158,32 +158,34 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
             initialScheduleListData = new ScheduleList();
         }
 
-        // Session List
+        // Log List
         try {
-            sessionListOptional = storage.readSessionList();
-            if (sessionListOptional.isEmpty()) {
+            logListOptional = storage.readLogList();
+            if (logListOptional.isEmpty()) {
                 logger.info("Data file not found. Will be starting with a sample ExerciseList");
             }
-            initialSessionListData = sessionListOptional.orElseGet(SampleSessionDataUtil::getSampleSessionList);
+            initialLogListData = logListOptional.orElseGet(SampleLogDataUtil::getSampleLogList);
 
             try {
-                storage.saveSessionList(initialSessionListData);
+                storage.saveLogList(initialLogListData);
```

Comment:
Jia chen you wanna move your saving code to execute method in LogicManager? We are all doing it there

Reply:
btw sorry I think I broke your code when I tried to resolve the merge conflicts, wrong import coz of the name change of session, can just revert that please

Reply:
I have settled the merge conflict! No worries bro! And good catch have moved the save logic in to the logic manager! Thanks!

Reply:
Verify that saving is done in logic manager so closed